### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by pviotti**
Currently, the README feels more useful to maintainers than to new contributors, and doesn't provide an overview of the project.  
This PR moves content from CONTRIBUTING.md to README.md and expounds on components, documentation and ways to contribute.  

The instructions to build OH packages are hidden away in a summary/details tag at the bottom of the readme, since it's not a primary interest of contributors.

(Roadmap link has been omitted because its content is not up-to-date)

Preview here: https://github.com/informatici/openhospital/blob/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
+
+- try the [desktop application][releases] or the early versions of the [web UI][ui]
+- request new features or report issues on [JIRA][jira] ([here][good-first]'s a list of *good-first-issues*)
+- improve the [documentation][doc]
+- contribute code patches to one of the components
+
+> NB: GitHub Issues are disabled on the repositories of individual components. 
+They are only enabled on this repository, with the purpose of collecting reports of bugs and 
+issues about this repository only. Instead, Jira is used to collect issues and plan 
+the development across all OH components.
```
Comment (mwithi) [inline README.md]: and also for end-users? Jira Service Desk is disabled at the moment, no plans to enable it

**Review by mwithi — APPROVED** [review]
LGMT, thanks!

Should we align README.md in other repos?

File: README.md (hunk 1 above)
Comment (pviotti) [reply README.md]: I'm a bit confused about the whole role of github issues vs jira issues, so I'll let you fix the phrasing as you prefer

**Comment by pviotti** [comment]
> Should we align README.md in other repos?

I already submitted PRs for most of the other repos in the past days. No need to give an broad overview of the project in every component's repo IMO.

**Pull request merged**